### ISSUE TITLE
Conference room topic in logs?

### ISSUE THREAD

**Issue by Vizis** [profanity-im/profanity#1889]
Is there any way to save conference topic in conference logs?

**Comment by jubalh** [comment]
So far not. You are speaking about the text chatlogs ~/.local/share/profanity/chatlogs ?

**Comment by Vizis** [comment]
yes, sorry for late response. Like when i use pidgin (xmpp client for windows) every time i connect to chat room aka conference i see the topic aka MOTD.

**Comment by jubalh** [comment]
Personally not interested in this.
